apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/RecoveryPhraseScreen.tsx

```diff
@@ -127,11 +127,13 @@ export const RecoveryPhraseScreen = () => {
       <View style={styles.content}>
         <StepperProgress currentStep={2} currentStepTitle={strings.stepRecoveryPhrase} totalSteps={4} />
 
-        <Text style={styles.title}>
+        <Text style={[styles.title, {lineHeight: 24}]}>
```

Comment:
https://emurgo.atlassian.net/browse/YOMO-1334

Getting an svg inline with text in react native is an ABSOLUTE nightmare

Reply:
styles.